controllers/workspace/condition.go

```diff
@@ -130,3 +132,12 @@ func getConditionIndexInOrder(condType dw.DevWorkspaceConditionType) int {
 	}
 	return -1
 }
+
+func capitalizeMsg(msg string) string {
```

Comment:
The go standard library actually has [cases](https://pkg.go.dev/golang.org/x/text/cases) for this use-case (handling unicode string capitalization).
Rather than re-invent the wheel, you could do something like:

```go
import (
	"golang.org/x/text/cases"
	"golang.org/x/text/language"
)
...

return cases.Title(language.English, cases.NoLower).String(msg)
```